src/core/Elsa.Abstractions/Events/ManyWorkflowDefinitionsDeleted.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using Elsa.Models;
+using MediatR;
+
+namespace Elsa.Events
+{
+    public class ManyWorkflowDefinitionsDeleted : INotification
```

Comment:
I'm pretty sure you didn't add this class of your own volition - might there be an issue with your PR, e.g. the fork not being up to date?

Reply:
My fork was up to date. I added `WorkflowDefinitionDeleted, WorkflowInstanceDeleted, WorkflowDefinitionSaved` and the class `EventPublishingWorkflowDefinitionStore`.  I added `EventPublishingWorkflowDefinitionStore` so that I could see changes to WorkflowDefinitions.

Reply:
Sorry, you are right. I got confused.